test: [M3-10366] - Add Cypress LKE-E 'phase2Mtc' feature flag smoke tests

mjac0bs opened the pull request:
## Description 📝

Add Cypress smoke tests to cover the lkeEnterprise feature flag phase2Mtc property's enabled and disabled states.

- When phase2Mtc is true, confirm that new LKE-E/VPC features are present
- When phase2Mtc is false, confirm that new LKE-E/VPC features are absent

## Changes  🔄
- Moves some shared utils and constants from `smoke-lke-create` spec to the constants and utils files for LKE
   -  Renames this file `smoke-lke-standard-create` to follow new naming convention
- Creates a new `smoke-lke-enterprise` spec:
   - The intent is to delete this spec file after the feature goes to GA and we unflag everything
   - The spec does basic page checks to confirm the new features are visible when the flag is on; else, hidden

### Scope 🚢

 Upon production release, changes in this PR will be visible to:

- [ ] All customers
- [ ] Some customers (e.g. in Beta or Limited Availability)
- [x] No customers / Not applicable

## Target release date 🗓️

9/9

## Preview 📷

<img width="847" height="638" alt="image" src="https://github.com/user-attachments/assets/fa7f47ea-2837-4b0a-a933-8af7b0eaa6cc" />

## How to test 🧪

### Verification steps

(How to verify changes)

```
pnpm cy:run -s "cypress/e2e/core/kubernetes/smoke-lke-enterprise.spec.ts"
```

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>

- [ ] I have read and considered all applicable items listed above.

## As an Author, before moving this PR from Draft to Open, I confirmed ✅

- [ ] All tests and CI checks are passing
- [ ] TypeScript compilation succeeded without errors
- [ ] Code passes all linting rules

</details>

<!-- This content will not appear in the rendered Markdown 

## Commit message and pull request title format standards

> **Note**: Remove this section before opening the pull request
**Make sure your PR title and commit message on squash and merge are as shown below**

`<commit type>: [JIRA-ticket-number] - <description>`

**Commit Types:**

- `feat`: New feature for the user (not a part of the code, or ci, ...).
- `fix`: Bugfix for the user (not a fix to build something, ...).
- `change`: Modifying an existing visual UI instance. Such as a component or a feature.
- `refactor`: Restructuring existing code without changing its external behavior or visual UI. Typically to improve readability, maintainability, and performance.
- `test`: New tests or changes to existing tests. Does not change the production code.
- `upcoming`: A new feature that is in progress, not visible to users yet, and usually behind a feature flag.

**Example:** `feat: [M3-1234] - Allow user to view their login history`

-->

jdamore-linode commented on packages/manager/cypress/e2e/core/kubernetes/smoke-lke-enterprise.spec.ts:
```diff
@@ -0,0 +1,254 @@
+    addNodes('Linode 2 GB');
+
+    // Bypass ACL validation
+    cy.get('input[name="acl-acknowledgement"]').check();
```
[TIL 💡](https://docs.cypress.io/api/commands/check). Neat!

jdamore-linode reviewed (approved):
Thanks Mariah, the tests look and run great!

mjac0bs commented:
> @mjac0bs I believe the CI failures are because the test accounts don't have the LKE enterprise tag so none of the regions have the LKE-E capability. Would it make more sense for these tests to mock the regions, or should I add the tag to the accounts?

Thanks, Joe. I added mocks for /account and /regions ([c789f70](https://github.com/linode/manager/pull/12773/commits/c789f707eca2d3eac198bd0a4b5f11e6dfb27099)) and removed the customer tag from my account to confirm that things should be passing in CI now.
